src/main/java/com/cburch/logisim/util/LineBuffer.java

```diff
@@ -501,27 +495,23 @@ public List<String> getWithIndent(String indent) {
     final var result = new ArrayList<String>();
     for (final var line : contents) {
       final var lines = line.split("\n");
-      for (var idx = 0; idx < lines.length; idx++) {
+      for (String s : lines) {
```

Comment:
Please use a more descriptive variable name.
e.g. `line`

Reply:
Also I believe `String` can be `var`'ed. Best if could also be `final`.

Reply:
Please make a seperate PR for this one

Reply:
PR #1509